pyglui/ui_elements.pxi

```diff
@@ -929,25 +938,51 @@ cdef class Thumb(UI_element):
             utils.draw_circle(self.button.center,radius=int(min(self.button.size)*.77), stroke_width = stroke_width, color=RGBA(*thumb_color_shadow),sharpness=thumb_button_shadow_sharpness)
             utils.draw_circle(self.button.center,radius=int(min(self.button.size)*.77), stroke_width = stroke_width, color=self.off_color,sharpness=thumb_button_sharpness)
 
+        print 'button center: ', self.button.center
+        print 'radius: ', int(min(self.button.size)*.77)
+        print 'stroke_width: ', stroke_width
```

Comment:
Use `logger.debug(<string>)` for consistent logging messages. `print` statements will always be displayed. Debug messages are only shown if the logger is configured that way.